### ISSUE TITLE
Move Transformers from Machines to Multiphase?

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#290]
**Reported by dietmarw on 15 Feb 2010 13:36 UTC**
When ever I was looking for the transformer models I ended up not finding them until I remembered that they are sitting in the `Modelica.Electrical.Machines.BasiMachines.Transformers` library. For me this is not a very logical placement. Intuitively I would rather look for them in `Modelica.Electrical.Multiphase.Transformers`.

I don't know what the reason for the current place was but maybe we can improve this for the next version with allowed conversions.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/290

**Comment by modelica-trac-importer** [comment]
**Comment by AHaumer on 15 Feb 2010 19:07 UTC**
Sorry if that sounds rude, Dietmar, but I won't do that.

MultiPhase is for arbitrary number of phases, 
Machines is limited to m=3 phases (technical threephase transformers).

MultiPhase is in parallel to Analog, the transformer in there is an ideal one without star- or polygon-connection, without stray inductances and winding resistances.
The transformers of the machine package are ready connected technical threephase transformers, they got a thermalPort, they'll get a core loss model and maybe more ...

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 16 Feb 2010 06:42 UTC**
No problem, you not being rude you just made your point. And although I'd like to see the transformers in a more logical place (as in easier to find), right now I can't really think of a better alternative. So closing as won't fix.

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 26 Oct 2010 06:15 UTC**
